SCClassLibrary/Common/Control/Server.sc

```diff
@@ -844,7 +911,9 @@ Server {
 	}
 
 	bootInit { | recover = false |
-		if(recover) { this.newNodeAllocators } { this.newAllocators };
+		// if(recover) { this.newNodeAllocators } {
+		// 	"% calls newAllocators\n".postf(thisMethod);
+		// this.newAllocators };
```

Comment:
why are these lines commented out?

Reply:
This is accidentally still commented out from testing. will uncomment, thanks for spotting!